interface_tester/collector.py

```diff
@@ -277,7 +277,7 @@ def gather_test_spec_for_version(
             "schema": schemas.get("requirer"),
             "charms": charms.get("requirers", []) if charms else [],
         },
-        "owner": charms.get("owner", "") if charms else "",
+        "owners": charms.get("owners", "") if charms else "",
```

Comment:
I think the defaults should be `[]` rather than `""` now. I often use `charms.get("owners") or []` instead of `charms.get("owners", [])` in case the field is present by null (not likely but still).

Reply:
Thanks for pointing it out! Good catch :) I have fixed in the latest commit.